test/test_complex_aliases.py

```diff
@@ -0,0 +1,60 @@
+import sqlparse
+
+from sql_metadata import Parser
+
+import pathlib
+
+dir_path = pathlib.Path(__file__).parent.absolute()
+
+
+def test_complex_query_aliases():
+    sql_filename = f"{dir_path}/test.sql"
+    with open(sql_filename, "r", encoding="latin-1") as content_file:
+        content = content_file.read()
+    content = sqlparse.format(content, strip_comments=True).strip()
+    parser = Parser(content)
```

Comment:
Maybe we can improve the `Parser` and allow it to consume either strings or file object?